sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/SdkDoubleHistogram.java

```diff
@@ -90,7 +95,18 @@ public LongHistogramBuilder ofLongs() {
     @Override
     public ExtendedDoubleHistogramBuilder setExplicitBucketBoundariesAdvice(
         List<Double> bucketBoundaries) {
-      adviceBuilder.setExplicitBucketBoundaries(bucketBoundaries);
+      double[] boundaries;
+      try {
+        Objects.requireNonNull(bucketBoundaries, "bucketBoundaries must not be null");
+        boundaries =
+            ExplicitBucketHistogramUtils.validateBucketBoundaries(
+                bucketBoundaries.stream().mapToDouble(value -> value).toArray());
```

Comment:
I'm was only going to an array here because `ExplicitBucketHistogramUtils#validateBucketBoundaries` requires it. But after looking at the code, there's really no reason for `ExplicitBucketHistogramUtils#validateBucketBoundaries` to need `double[]` over `List<Double>`, so I've adjusted the signature and simplified the code in these builders.